frontend/src/visualization/Visualization.tsx

```diff
@@ -0,0 +1,45 @@
+import { Box, Container, Grid } from '@material-ui/core';
+import React from 'react';
+import routes from '../routing/routes';
+import DashboardCard from './dashboard-card/DashboardCard';
+import createCard from './dashboard-card/cardContents';
+
+function Visualization(): JSX.Element {
+  const { tabs } = routes.Visualization;
+
+  if (tabs === undefined) {
+    throw new Error('Cannot read Visualization tabs');
```

Comment:
Nevermind Linter says no console
>In JavaScript that is designed to be executed in the browser, it's considered a best practice to avoid using methods on console. Such messages are considered to be for debugging purposes and therefore not suitable to ship to the client. In general, calls using console should be stripped before being pushed to production.
I'll change it to `return <></>;`

Reply:
`console.error` freaks out the linter. Just ignore it?

Reply:
@derwehr You already found out about the linter. Maybe I should read all comments before answering.